manual.md

```diff
@@ -1518,12 +1606,22 @@ Graphic and Sound Statements
   Missiles share the same color as
   their player, unless you combine
   them into a "5th Player" by setting
-  the 5th bit of the PRIOR register,
+  the 5th bit of the `GPRIOR` register,
   e.g.: `POKE 623,16`. (You must also
   move them horizontally in unison if
   you wish to use them as a true
   5th Player.)
 
+  It is possible to cause pixels of
+  certain overlapping players to
+  produce a third color (or black)
+  by setting the 5th bit of the
```

Comment:
It is the 6th bit, or bit number 5.... for consistency with the above. But perhaps it is better to replace both the above and this to "the bit number 4" and "the bit number 5" respectively.